config/postcreate-hooks/kubestellar.yaml

```diff
@@ -184,7 +184,7 @@ spec:
               - kubestellar
               - oci://ghcr.io/kubestellar/kubestellar/controller-manager-chart
               - --version
-              - "0.20.0-alpha.1"
+              - "v0.20.0-rc1"
```

Comment:
Are we going to manually change this on every release ?  Can't we `sed` it or something through the Make ?

Reply:
@ezrasilvera we do not have a process for that for now. We discussed several approaches (including publishing a helm chart with the release). But for the time being it needs to have the same tag of the release being published so it can be referenced later in the examples after testing the RC and updating the examples. Since this is a larger issue with PCHs I suggest to leave this for successive PRs.